src/include/common/types/value.h

```diff
@@ -124,27 +258,63 @@ class NodeVal {
     std::vector<std::pair<std::string, std::unique_ptr<Value>>> properties;
 };
 
+/**
+ * @brief RelVal represents a rel in the graph and stores the relID, src/dst nodes, properties of
```

Comment:
"the relID, src/dst nodes, properties of" -> "the relID, src/dst nodes, and properties of"